src/react/LoginCallBack.js

```diff
@@ -25,27 +25,19 @@ const LoginCallBack = ( { successCallback, errorCallback, customErrorComponent,
             {
                 try
                 {
-                    const authState = await ocAuth.handleLoginRedirect();
-                    if ( authState.idToken )
+                    await ocAuth.handleLoginRedirect();
+                    if ( ocAuth.isAuthenticated() )
```

Comment:
ah ok -> Re my comments in the `OCConnect`. 
I believe
if you follow my comments there and have dependency array on `isAuthenticated` you won't even need it here. 
you don't rely on this component to call `updateAuthState` but let the `useEffect` in OCConnect to trigger the code that update the authstate state variable.